Lib/test/test_pyexpat.py

```diff
@@ -755,5 +755,73 @@ def resolve_entity(context, base, system_id, public_id):
         self.assertEqual(handler_call_args, [("bar", "baz")])
 
 
+class ReparseDeferralTest(unittest.TestCase):
+    def test_getter_initial_value(self):
```

Comment:
This is already covered by the following test.

Reply:
@serhiy-storchaka alright, dropped